### ISSUE TITLE
On the use of variable stride in convolutional neural networks

### ISSUE THREAD

**Issue by linahourieh** [linahourieh/machine-learning-articles#11]
## TL;DR
This paper explores the idea of changing the stride value in convolutional neural networks
depending on the position of the pixel within the image: a smaller stride value is used
when processing the center of the image, while a larger one is used for pixels close to the
edges
### Article Link
[s11042-019-08385-4.pdf](https://github.com/linahourieh/machine-learning-articles/files/9546025/s11042-019-08385-4.pdf)

### Author
Luiz Zaniolo · Oge Marques
## Key Takeaways
* smaller strides --> central pixels
* larger strides --> edge pixels
* Sometimes varying the strides outperforms the fixed stride strategy in terms of accuracy, computational cost, inference time.
* the smaller the stride, the more computations are needed and the bigger is the resulting output

![Screenshot from 2022-09-12 10-09-50](https://user-images.githubusercontent.com/81252980/189604263-da904602-cbca-4990-b3b6-9c2858ff79ed.png)
![Screenshot from 2022-09-12 10-10-22](https://user-images.githubusercontent.com/81252980/189604354-09fb121f-dc80-4fa2-bacf-b34e8b04716b.png)

* The use of variable stride can improve the performance of CNN-based image classification solutions **for cases** where most of the semantic information associated with the image resides around the central portion of the image.
